WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -136,11 +141,23 @@ final class DashboardViewModel: ObservableObject {
     func reloadAllData() async {
         await withTaskGroup(of: Void.self) { group in
             group.addTask { [weak self] in
-                guard let self else { return }
-                await self.syncAnnouncements(for: self.siteID)
+                await self?.syncDashboardEssentialData()
+            }
+            group.addTask { [weak self] in
+                guard let self = self else { return }
+                await self.reloadCards(self.showOnDashboardCards)
```

Comment:
Super nit:

```suggestion
                guard let self else { return }
                await reloadCards(showOnDashboardCards)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/12775/commits/df5b2fc329d152f9a0416342789fa343f05f7fc8